draft-bbs-signatures.md

```diff
@@ -392,57 +400,57 @@ Procedure:
 
 1. (A, e, s) = signature
 
-2. (w, h0, h\[1\],...,h\[L\]) = PK
+2. (W, H0, H[1],...,H[L]) = PK
 
 3. (i1, i2,..., iR) = RIdxs
 
-4. if subgroup\_check(A) is INVALID abort
+4. if subgroup_check(A) is INVALID abort
 
 5. if KeyValidate(PK) is INVALID abort
 
-6. b = commitment + h0 \* s + h\[1\] \* msg\[1\] + ... + h\[L\] \* msg\[L\]
+6. B = commitment + H0 * s + H[1] * msg[1] + ... + H[L] * msg[L]
 
-7. r1 = H(PRF(8\*ceil(log2(r)))) mod r
+7. r1 = HASH(PRF(8*ceil(log2(q)))) mod q
 
-8. r2 = H(PRF(8\*ceil(log2(r)))) mod r
+8. r2 = HASH(PRF(8*ceil(log2(q)))) mod q
 
-9. e\~ = H(PRF(8\*ceil(log2(r)))) mod r
+9. e~ = HASH(PRF(8*ceil(log2(q)))) mod q
 
-10. r2\~ = H(PRF(8\*ceil(log2(r)))) mod r
+10. r2~ = HASH(PRF(8*ceil(log2(q)))) mod q
 
-11. r3\~ = H(PRF(8\*ceil(log2(r)))) mod r
+11. r3~ = HASH(PRF(8*ceil(log2(q)))) mod q
 
-12. s\~ = H(PRF(8\*ceil(log2(r)))) mod r
+12. s~ = HASH(PRF(8*ceil(log2(q)))) mod q
 
-13. r3 = r1 ^ -1 mod r
+13. r3 = r1 ^ -1 mod q
 
-14. for i in RIdxs m\~\[i\] = H(PRF(8\*ceil(log2(r)))) mod r
+14. for i in RIdxs m~[i] = HASH(PRF(8*ceil(log2(q)))) mod q
 
-15. A' = A \* r1
+15. A' = A * r1
 
-16. Abar = A' \* -e + b \* r1
+16. Abar = A' * -e - B * r1
```

Comment:
`+ B * r1` I believe

Reply:
Based on `Abar = A'k` checked by the pairing. `Abar = A'(-e) + B*r1 = A'(-e) + A'(e + k) = A'k`

Reply:
It matches all the other changes I've made.